docs/JestObjectAPI.md

```diff
@@ -141,7 +141,9 @@ module.exports = {
 ```js title="__tests__/createMockFromModule.test.js"
 const utils = jest.createMockFromModule('../utils');
 
-utils.isAuthorized = jest.fn(secret => secret === 'not wizard');
+jest
+  .spyOn(utils, 'isAuthorized')
+  .mockImplementation(secret => secret === 'not wizard');
```

Comment:
it doesn't make sense to `spyOn` here - `isAuthorized` is already a mock function (that's what `createMockFromModule` does)

Reply:
Ups.. This file should not be here. My bad (;

Reply:
instead of `utils.isAuthorized = jest.fn(secret => secret === 'not wizard');` it should probably be `utils.isAuthorized. mockImplementation(secret => secret === 'not wizard');`

Reply:
These lines got autocorrected, because I was playing with `jest/prefer-spy-on` ESLint rule. I was hoping to find better example for docs, but nothing came out. `jest/prefer-spy-on` rule will come in later as separate PR (;